### ISSUE TITLE
Enabling tracing in Android for long package names

### ISSUE THREAD

**Issue by tonylombardi** [apitrace/apitrace#296]
Following the instructions in Dalvik.markdown, it's necessary to set a property in order to trace an app in Android

"PROCNAME=com.android.settings
adb root
adb shell setprop wrap.$PROCNAME LD_PRELOAD=/data/egltrace.so"

But Android limits properties to 31 characters max.  Many apps have package names too long.  Is there a workaround for this?

**Comment by stefand** [comment]
Simply truncating worked for me, fwiw.

**Comment by jrfonseca** [comment]
@stefand, thanks.

I've added a note about this in the docs.

**Comment by enovella** [comment]
Hi @stefand, @jrfonseca and @amonakov,

I used to use this wrapping property on Android KitKat devices 4.4.4 and my shared object was indeed preloaded. However, I have found the issue of the maximum length in the application name that apparently it needs to be truncated to work properly. 

In my experience on Android 7.x and the same library but different application name (which needs to be truncated) does not overload any on my preloaded functions.

Any ideas how to overcome that issue?

Thanks in advance guys. Cheers
